database/data_seeding/data_generator.js

```diff
@@ -92,6 +92,51 @@ const generateNoSqlData = (batchSize, startingId) => {
 const generateSqlData = (batchSize, startingId) => {
   const data = [];
 
+  while (batchSize > 0) {
+    const productFeaturesDataObject = {
+      featuresList: [],
+      contentGridFeatureItems: [],
+    };
+
+    let numFeatureListItems = 7;
+    let numContentGridItems = 5;
+
+    productFeaturesDataObject.feature = {
+      id_decid: startingId,
+      feature_banner_header: `'${fakeHeader()}'`,
+      feature_banner_text_1: `'${fakeDescription()}'`,
+      feature_banner_text_2: `'${fakeDescription()}'`,
+      feature_setup_header: `'${fakeHeader()}'`,
+      feature_setup_description_1: `'1. ${fakeSentence()}'`,
+      feature_setup_description_2: `'2. ${fakeSentence()}'`,
+      feature_setup_description_3: `'3. ${fakeSentence()}'`,
+      additional_features_header: `'${fakeHeader()}'`,
+      additional_features_description: `'${fakeDescription()}'`,
```

Comment:
these functions (fakeHeader, fakeSentence, fakeDescription), do they not return a string? Is it necessary to put them in a template literal or can you just call the function so it would look more like this:
additional_features_description: fakeDescription() as opposed to `'${fakeDescription()}'`?

Reply:
They do return a string exactly, however (if I'm not mistaken), I still need to format them with single ticks around them for SQL syntax/formatting purposes. That's at least how I've done it before...